packages/functions/src/index.ts

```diff
@@ -16,6 +16,41 @@ import {logger} from "firebase-functions";
 
 admin.initializeApp();
 
+export const scheduledEventStatusUpdate = functions.pubsub
+  .schedule("every 240 hours")
```

Comment:
I suppose here will be better use constant

Reply:
Done